apps/console/src/features/users/components/parent-user-invite-response-list.tsx

```diff
@@ -0,0 +1,310 @@
+                        <Header as="h6" data-componentid={ `${componentId}-username-item-heading` }>
+                            <Header.Content>
+                                { item.username }
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.username")
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "status",
+                id: "status",
+                key: "status",
+                render: (item: ParentUserInviteResponseListItem): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}-status-item-heading` } >
+                            <Header.Content>
+                                <Label
+                                    data-componentid={ `${componentId}-bulk-label` }
+                                    content={ item.status }
+                                    size="small"
+                                    color={ item.status === ParentOrgUserInviteResultStatus.SUCCESS ? "green" : "red" }
+                                    className="group-label"
+                                />
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.status")
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "message",
+                id: "message",
+                key: "message",
+                render: (item: ParentUserInviteResponseListItem): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}-status-item-heading` }>
+                            <Header.Content>
+                                { item.message }
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.message")
+            }
+        ];
+
+        return defaultColumns;
+    };
+
+    /**
+     * Shows list placeholders.
+     *
+     * @returns Placeholders.
+     */
+    const showPlaceholders = (): ReactElement => {
+        return (
+            <EmptyPlaceholder
+                data-testid={ `${componentId}-empty-search-result` }
+                data-componentid={ `${componentId}-empty-search-result` }
+                image={ getEmptyPlaceholderIllustrations().emptySearch }
+                imageSize="tiny"
+                title={ t("console:manage.features.users.usersList.search.emptyResultPlaceholder.title") }
+                subtitle=""
```

Comment:
Is setting subtitle to empty string intentional?

Reply:
Yes